qoqo/tests/integration/circuit.rs

```diff
@@ -412,6 +413,20 @@ fn test_to_from_json() {
     })
 }
 
+/// Test json_schema function of Circuit
+#[cfg(feature = "json_schema")]
+#[test]
+fn test_json_schema() {
+    pyo3::prepare_freethreaded_python();
+    pyo3::Python::with_gil(|py| {
+        let circuit = new_circuit(py);
```

Comment:
I'd recommend adding an operation to the circuit for additional testing, but that may just be personal preference